lib/orderbook/MatchingEngine.ts

```diff
@@ -184,18 +184,8 @@ class MatchingEngine {
     return removedOrders;
   }
 
-  /**
-   * Removes an own order by its global order id.
-   * @returns the removed order, or undefined if no order with the provided id could be found
-   */
-  public removeOwnOrder = (orderId: string): StampedOwnOrder | undefined => {
-    const order = this.ownOrders.buy.get(orderId) || this.ownOrders.sell.get(orderId);
-    if (!order) {
-      return;
-    }
```

Comment:
1) to make it consistent with `removePeerOrder`. 2) An equivalent check happens after calling `getOwnOrder` - OrderBook.ts:378 - which checks to make sure that the order we're trying to remove actually exists.

Reply:
The equivalent check you have in `OrderBook` requires calling `getOwnOrder`. I'd rather make it consistent with `removePeerOrder` by adding this check to the latter, instead of removing it from `removeOwnOrder`.